content/concepts/loaders.md

```diff
@@ -1,6 +1,163 @@
 ---
 title: Loaders
 sort: 2
+contributors:
+  - manekinekko
 ---
-> module.loaders configuration option
-> test, include, exclude, loader, query
+
+## What are loaders?
+
+Loaders are transformations that are applied on a resource file of your app. They are functions (running in node.js) that take the source of a resource file as the parameter and return the new source.
+
+For example, you can use loaders to tell webpack to load TypeScript or JSX.
+
+### Loader features
+
+* Loaders can be chained. They are applied in a pipeline to the resource. The final loader is expected to return JavaScript; each other loader can return source in arbitrary format, which is passed to the next loader.
+* Loaders can be synchronous or asynchronous.
+* Loaders run in Node.js and can do everything that's possible there.
+* Loaders accept query parameters. This can be used to pass configuration to the loader.
+* Loaders can be bound to extensions / RegExps in the configuration.
+* Loaders can be published / installed through `npm`.
+* Normal modules can export a loader in addition to the normal `main` via `package.json` `loader`.
+* Loaders can access the configuration.
+* Plugins can give loaders more features.
+* Loaders can emit additional arbitrary files.
+* etc.
+
+If you are interested in some loader examples head off to the [[list of loaders]].
```

Comment:
There is no list of loaders in the current documentation so it's probably leave this mention out.